apps/jenkins/jenkins/ptl-intsvc/jenkins.yaml

```diff
@@ -340,7 +340,7 @@ spec:
                     title: FinRem Dashboard
                 - buildMonitor:
                     includeRegex: >-
-                      ^HMCTS.*\/(et-ccd-callbacks|et-message-handler|et-shared-infrastructure|et-sya-frontend|et-sya-api|et-xui-e2e-tests|et-ccd-definitions-[a-z]+)\/master
+                      ^HMCTS.*\/(et-ccd-callbacks|et-message-handler|et-shared-infrastructure|et-sya-frontend|et-sya-api|et-xui-e2e-tests|et-ccd-definitions|et-hearings-api-[a-z]+)\/master
```

Comment:
The definitions needed the ' -[a-z] ' because there are multiple ones, you need the et-hearings-api to be in a separate alternative

Reply:
Thanks very much makes sense have adjusted as suggested